test/run_tests.sh

```diff
@@ -152,7 +166,7 @@ function run_op_tests {
   run_save_tensor_file python3 "$CDIR/dynamo/test_dynamo_graph_dump.py"
   run_downcast_bf16 python3 "$CDIR/test_data_type.py"
   run_use_bf16 python3 "$CDIR/test_data_type.py"
-  run_test python3 "$CDIR/test_torch_distributed_xla_backend.py"
+  # run_test python3 "$CDIR/test_torch_distributed_xla_backend.py"
```

Comment:
ditto.

Reply:
disable test_torch_distributed_xla_backend.py because of https://pantheon.corp.google.com/cloud-build/builds/6f8383e1-f7b0-4bab-bd24-2e19adaa8523?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override

Reply:
same, we should not disable this for CPU and GPU.

Reply:
Thanks, Jack, updated